src/channel.ts

```diff
@@ -798,6 +738,18 @@ export class Channel<
     members: string[],
     message?: Message<AttachmentType, MessageType, UserType>,
   ) {
+    return await this._update({
+      demote_moderators: members,
+      message,
+    });
+  }
+
+  /**
+   * _update - executes channel update request
+   * @param payload Object Update Channel payload
+   * @return {Promise<UpdateChannelAPIResponse<AttachmentType, ChannelType, CommandType, MessageType, ReactionType, UserType>>} The server response
+   */
+  async _update(payload: Object) {
```

Comment:
We don't have type for this. Do we need it?

Reply:
generally yes, since the entire lib is in TS, private functions need to be also typed to make everyone lifes easier

Reply:
I tried that and there are some problems that I'm not sure if we want to address now.

1. We already have `InviteOptions` type which should be called `ChannelUpdateOptions` or `UpdateChannelOptions`. We can create an alias to keep BC, but...
2. We also have `UpdateChannelOptions` which should be called `UpdateChannelTypeOptions`

I would suggest we resolve this naming conflict in the next major release and only then introduce strict type here. Introducing new type now will only add confusion

Reply:
In that case, it's not an urgent fix for now, could you leave a TODO there?
let's pull @nhannah to be in the loop as well